src/script.c

```diff
@@ -6,6 +6,7 @@
 #include "http_parser.h"
 #include "stats.h"
 #include "zmalloc.h"
+#include "sys/time.h"
```

Comment:
Since `sys/time.h` is a system header, I think it would be "more correct" to use
`#include <sys/time.h>` in this case

Reply:
On top of @mseri comments, it should be moved to above line 5.